README.md

```diff
@@ -78,11 +78,14 @@ mvn clean install
 Go to your `podium-registry` folder and start the registry with `mvn`.
 The user interface of the registry will be available at [http://localhost:8761](http://localhost:8761).
 
-2. **Start Podium UAA**<br>
+2. **Start ElasticSearch**
+Run `docker run -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.8.15`.
```

Comment:
To prevent having these random generated container names:
```suggestion
Run `docker run --name podium-elasticsearch -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" docker.elastic.co/elasticsearch/elasticsearch:6.8.15`.
```